templates/scitldr/Abstract/templates.yaml

```diff
@@ -0,0 +1,41 @@
+dataset: scitldr
+subset: Abstract
+templates:
+  01fb91ab-2c95-436e-9363-3dfcdb6c5ba6: !Template
+    id: 01fb91ab-2c95-436e-9363-3dfcdb6c5ba6
+    jinja: "Generate a summary for the text: \n{{source | join(\" \")}}\n|||\n{{target[0]}}"
+    name: basic - task description like
+    reference: Assume there is only one choice
+    task_template: true
+  08b9e913-a305-46e2-aa43-f1126d76cf55: !Template
+    id: 08b9e913-a305-46e2-aa43-f1126d76cf55
+    jinja: "Elaborate the given summary: \n{{target |choice}}\n\nStart with following\
```

Comment:
It think it's more grammatical to say "elaborate on" here.

Reply:
Resolved!